### PR TITLE
Fix release branch

### PR DESCRIPTION
**Pull request by 0xdavinchee**

### CODE REVIEW

**Comment by 0xdavinchee** [comment]
should merge to dev, then will cherry pick into release branch if all is well

File: .github/workflows/handler.publish-release-packages.yml [hunk 1]
```diff
@@ -43,22 +43,23 @@ jobs:
...
+          tasks/npm-publish.sh packages/ethereum-contracts/ latest
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPMJS_TOKEN: ${{ secrets.NPMJS_TOKEN }}
 
       - name: Publish js-sdk@latest package
         if: env.PUBLISH_JS_SDK == 1
-        run: tasks/npm-publish.sh packages/js-sdk/ latest
+        run: |
+          yarn --cwd packages/js-sdk build
```
Comment (hellwolf) [inline .github/workflows/handler.publish-release-packages.yml:62]: unfortunately js-sdk build requires ethereum-contracts build.

File: .github/workflows/handler.publish-release-packages.yml [hunk 2]
```diff
@@ -107,6 +110,7 @@ jobs:
         id: publish-sdk-redux
         if: env.PUBLISH_SDK_REDUX == 1
         run: |
+          yarn --cwd packages/sdk-redux build
```
Comment (hellwolf) [inline .github/workflows/handler.publish-release-packages.yml:115]: the build dependencies start to become harder to reason now with these code, not sure if there is a good solution other than write build dependencies explicitly here

Comment (0xdavinchee) [reply .github/workflows/handler.publish-release-packages.yml:115]: yeah, I agree
SDK-Core and JS-SDK both need contracts built, looking at sdk-redux and it looks like it doesn't have any build dependencies

File: .github/workflows/handler.publish-release-packages.yml (hunk 1 above)
Comment (0xdavinchee) [reply .github/workflows/handler.publish-release-packages.yml:62]: resolved

**Review by hellwolf — APPROVED** [review]

**Pull request merged**